docker/k8s/fabric/README.md

```diff
@@ -61,8 +61,10 @@ This serves as a playground to setup k8s cluster for Avalon Fabric proxy model.
 3. To execute a transaction from shell
    ```bash
    cd examples/apps/generic_client
-   ./fabric_generic_client.py -b fabric --workload_id "echo-result" --in_data "Hello"
+   ./fabric_generic_client.py -b fabric --workload_id "echo-result" --in_data "Hello" \
+       --worker_id "singleton-worker-1"
    ```
+   NOTE: `worker_id` is the same value passed in `$TCF_HOME/config/singleton_enclave_config.toml`
```

Comment:
It's not necessary to keep worker id same as that in config file. You need to mention that worker id can either be passed as a command line parameter and if worker id is not passed in command line then worker id value in config toml file will be used. Command line parameter takes precedence over that mentioned in config file.

The generic client needs to choose a worker id which matches with worker id of singleton enclave manager worker

Reply:
Done